.github/CODEOWNERS

```
@@ -0,0 +1,237 @@
+/.github/ @elastisys/goto-pipeline/qa/release
```

Permission wise, all should be fixed. However these needs do be fixed before we can merge

```suggestion
/.github/ @elastisys/goto-pipeline-qa-release
```